patterns/2-structured/innersource-customer-interview-questions.md

```diff
@@ -0,0 +1,74 @@
+## Title
+
+InnerSource Customer Interview Questions
```

Comment:
Any chance we can leave "InnerSource" out of the title?
We are frequently at risk of having this word in every pattern title, which then removes meaning from the word. So we try to use it less frequently if possible.

Reply:
Clare had told me to add it in there on this one. Iorgiginally didn't have it in the title

Reply:
I see. @claredillon anything you can add here about your reasoning around the title of this pattern?

FYI the title is less of a pressing issue, now that this pattern is in status "Initial".
We can work this out some time later.